apps/src/feedback.js

```diff
@@ -387,6 +397,9 @@ FeedbackUtils.prototype.calculateStageProgress = function (hintsUsed) {
     numZeroHints = 0,
     numOneHint = 0;
   for (let i = 0; i < levels.length; i++) {
+    if (levels[i].ids.indexOf(currentLevelId) !== -1 ) {
+      continue;
+    }
```

Comment:
I wonder if this is a sign that we should be exposing a cleaner interface on `ClientState`.  Are there other places that would want to get the list of passed or perfected dots, collapsing level variants into a single best result?

Feel free to ignore, this shouldn't block check-in for this experiment.